client/src/modules/user/UserProfile/DeleteDialog.jsx

```diff
@@ -34,13 +36,15 @@ const DeleteDialog = ({
       <AlertBox type={MessageType.ERROR}>
         <FormattedMessage id="user.delete-account-warning" />
       </AlertBox>
-      {error && (
+      {error && !errorData && (
         <ErrorMessage
           message={formatMessage({ id: 'user.delete-account-error' })}
         />
       )}
+      {errorData && <ResourcePanel errorData={errorData} />}
```

Comment:
Please wrap them both within `{error && <>{...}</>}` section.

Reply:
Not every error comes with errorData.